ignite/pkg/cosmoscmd/root.go

```diff
@@ -382,7 +382,7 @@ func (a appCreator) newApp(
 		baseapp.SetIndexEvents(cast.ToStringSlice(appOpts.Get(server.FlagIndexEvents))),
 		baseapp.SetSnapshot(snapshotStore, snapshotOptions),
 		baseapp.SetIAVLCacheSize(cast.ToInt(appOpts.Get(server.FlagIAVLCacheSize))),
-		baseapp.SetIAVLDisableFastNode(cast.ToBool(appOpts.Get(server.FlagIAVLFastNode))),
+		baseapp.SetIAVLDisableFastNode(cast.ToBool(appOpts.Get(server.FlagDisableIAVLFastNode))),
```

Comment:
Can you point where does this come from ?

Because I remember we added the red line for a recent cosmos bug fix, also, and now we somehow disable it right ?

Reply:
This flag was introduced in sdk `v0.46.3`.  But then they changed the name of the flag.  

`FlagIAVLFastNode` -> `FlagDisableIAVLFastNode`.  So this change is just propagating that.